worlds/animal-well/names/item_names.py

```diff
@@ -61,3 +61,4 @@
 ball_tricky = "Ball Tricky"  # for hitting things with the ball that aren't obviously meant to be
 two_keys = "Two Keys"  # there's one spot that has two key doors adjacent to each other
 tanking_damage = "Tanking Damage"  # for spots you can get to by taking up to 3 hearts of damage
+weird_skips = "Weird Skips"  # skips that are questionably in logic and may or may not constitute a difficulty setting later
```

Comment:
```suggestion
weird_skips = "Weird Skips"  # skips that are questionably in logic and may or may not constitute a difficulty setting later

```
Pep8 says newline at end of file is needed